.golangci.yml

```diff
@@ -1,6 +1,4 @@
 run:
-  modules-download-mode: readonly
```

Comment:
??

Reply:
That had fixed an error I was having running the linter. It's been a while, so I can't remember what the error was.

Testing again with this line re-added doesn't cause a problem anymore. I wonder if it only breaks if your go modules aren't already up-to-date?

In any case, not really in scope for this PR, so reverting.